maven-core/src/test/resources/META-INF/plexus/components.xml

```diff
@@ -1,107 +0,0 @@
-<component-set>
```

Comment:
That's unrelated right ? It's just a left over from a previous commit ?

Reply:
Yes, but frankly, this XML was unused even before previous commit, it was simply a piece of garbage that I spotted when I went thru all Plexus XMLs